src/language-js/print/array.js

```diff
@@ -125,4 +136,32 @@ function printArrayItems(path, options, printPath, print) {
   return printedElements;
 }
 
+function printArrayItemsUsingFill(path, options, printPath, print) {
+  const parts = [];
+
+  path.each((childPath, i, elements) => {
+    const isLast = i === elements.length - 1;
+
+    parts.push([print(childPath), isLast ? "" : ","]);
+
+    if (!isLast) {
+      const element = elements[i];
+      const nextElement = elements[i + 1];
+      parts.push(
+        element &&
```

Comment:
nit: i would make the left half of the ternary
a constant, like you have `isLast` and
`shouldUseFill` above. That way you can
come up with a long identifier name that
makes it crystal clear to the reader what
the condition is, like youve done for the other 2.

Reply:
I've changed that logic. Now the condition looks the same as in the existing function `printArrayItems`.